From 49d0d39da9df96b2cb9d8a55880bb80d7568573e Mon Sep 17 00:00:00 2001
From: Johnny Mnemonic <jm@machine-hall.org>
Date: Tue, 25 Mar 2025 09:44:29 +0100
Subject: [PATCH 64/94] ia64/crash: Use note name macros

Use note name macros to match with the userspace's expectation.

See:
* d4a760fb77fdac07efa3da4fa4a18f49f178d048
* 609c8b30915697b3d520824b19b7f3a69eeda2c5
* 4f773fcbdb4698175ca908d332a1f7f3d4c8e93c (merge commit)
---
 arch/ia64/kernel/crash.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/ia64/kernel/crash.c b/arch/ia64/kernel/crash.c
index 65b0781f83ab..1d46d8acac1b 100644
--- a/arch/ia64/kernel/crash.c
+++ b/arch/ia64/kernel/crash.c
@@ -56,7 +56,7 @@ crash_save_this_cpu(void)
 	buf = (u64 *) per_cpu_ptr(crash_notes, cpu);
 	if (!buf)
 		return;
-	buf = append_elf_note(buf, KEXEC_CORE_NOTE_NAME, NT_PRSTATUS, prstatus,
+	buf = append_elf_note(buf, NN_PRSTATUS, NT_PRSTATUS, prstatus,
 			sizeof(*prstatus));
 	final_note(buf);
 }
-- 
2.25.1

